robot-server/robot_server/protocols/router.py

```diff
@@ -92,6 +94,22 @@ class ProtocolUsedByRun(ErrorDetails):
     title: str = "Protocol Used by Run"
 
 
+class RunLink(BaseModel):
+    """Link to a run resource."""
+
+    id: str = Field(..., description="The run's id")
+    href: str = Field(..., description="The run's URL")
+
+
+class ProtocolLinks(BaseModel):
+    """Links returned along with a protocol resource."""
+
+    referencingRunIds: List[RunLink] = Field(
+        [],
```

Comment:
```suggestion
        ...,
```

Instead of defaulting to the empty list, I would have no default (requiring the caller to always provide an explicit value), for two reasons:

1. It's probably a mistake if server code constructs a `ProtocolLinks` and forgets to provide a `referencingRunIds` list.
2. In the OpenAPI spec, FastAPI interprets response fields with defaults as "not required," even though the server *should* always provide them.